From 2f14b99f6f10510d194cb40efad0a243690ffea2 Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Thu, 30 Dec 2021 06:57:09 +0200
Subject: [PATCH 01/31] loosen requirements

* Remove unused libclang requirement
* Remove unecessary tensorflow-io-gcs-filesystem
* numpy version was unpinned in a previous patch,
  because of an upper bound. Recently it was removed

Co-authored-by: Mark Harfouche <mark.harfouche@gmail.com>
Co-authored-by: ngam
Co-authored-by: Uwe L. Korn <uwelk@xhochy.com>
Co-authored-by: Isuru Fernando <isuruf@gmail.com>
---
 tensorflow/tools/pip_package/setup.py.tpl | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tensorflow/tools/pip_package/setup.py.tpl b/tensorflow/tools/pip_package/setup.py.tpl
index 949dc393c23..21d3a9330e5 100644
--- a/tensorflow/tools/pip_package/setup.py.tpl
+++ b/tensorflow/tools/pip_package/setup.py.tpl
@@ -84,7 +84,6 @@ REQUIRED_PACKAGES = [
     'flatbuffers >= 24.3.25',
     'gast >=0.2.1,!=0.5.0,!=0.5.1,!=0.5.2',
     'google_pasta >= 0.1.1',
-    'libclang >= 13.0.0',
     'opt_einsum >= 2.3.2',
     'packaging',
     'protobuf>=5.28.0',
